tests/requests/test_views.py

```diff
@@ -1,10 +1,48 @@
 import pytest
 from django.urls import reverse
-
 from django_webtest import DjangoTestApp
 
 from vitrina.datasets.factories import DatasetFactory
 from vitrina.requests.factories import RequestFactory
+from vitrina.requests.models import Request
+from vitrina.users.models import User
+
+
+@pytest.mark.django_db
+def test_request_create(app: DjangoTestApp):
+    form = app.get(reverse("request-create")).forms[0]
+    form['title'] = "Request"
+    form['description'] = "Description"
+    resp = form.submit()
+    assert Request.objects.count() == 1
+    assert resp.status_code == 302
+    assert resp.url == reverse('request-detail', args=[Request.objects.first().pk])
```

Comment:
```suggestion
    assert resp.url == Request.objects.filter(title='Request').first().get_absolute_url()
```

Lets be consistent and always get object details page using `get_absolute_url`.